.github/pull_request_template.md

```diff
@@ -6,12 +6,13 @@ Please write a few sentences describing the overall goals of the pull request's
 
 ## PR checklist
 
-Please tick as appropriate and edit the reasons (e.g.: "backport: not needed because this is a new feature")
+Please remove add/remove strikethrough ~~ as appropriate, and add any relevant link/s to the end of the line.
 
-- [ ] **changelog** provided, or not required
-- [ ] **3.6 backport** done, or not required
-- [ ] **2.28 backport** done, or not required
-- [ ] **tests** provided, or not required
+- [ ] **changelog** provided, ~~not required~~, ~~covered~~ . Link --> #XYZ
+- [ ] **dev PR** provided, ~~not required~~ Link --> #XYZ
```

Comment:
The purpose of the `Link --> #XYZ` was to make clear to the user what its meant to do. I think `link: #` would be a good compromise, and after people are confident with it we cna remove the link part. 

The tool does not care since it will be looking for markdown http or #{0-9} type of linkage in the PR's body anyway. 

I can change the order of items like not `required, provided`, and `note required, covered provided` for changelog entries, but I do not think it should matter that much,